contracts/staking/SimpleStaking.sol

```diff
@@ -233,6 +231,8 @@ contract SimpleStaking is AbstractGoodStaking, StakingToken {
 				10**decimalDifference
 			);
 		}
+
+		tokenGains = getTokenPriceInUSD(tokenGains);
```

Comment:
need to update docs, you return value in usd while docs say value in token.
go over docs in general

Reply:
changed doc of it